build.sbt

```diff
@@ -6,7 +6,7 @@ organization in Global := "edu.gemini.ocs"
 
 ocsVersion in ThisBuild := OcsVersion("2016A", true, 1, 1, 1)
 
-pitVersion in ThisBuild := OcsVersion("2016A", false, 1, 1, 0)
+pitVersion in ThisBuild := OcsVersion("2016B", true, 2, 1, 0)
```

Comment:
Just out of curiosity: why will the version numbering for 2016B start at 2.1 and not 1.1?